console/services/backup_service.py

```diff
@@ -131,6 +131,9 @@ def backup_group_apps(self, tenant, user, region_name, group_id, mode, note, for
                 raise ServiceHandleException(msg="backup failed", msg_show="上次备份任务未完成或有正在恢复的备份或该版本已存在", status_code=409)
             if e.status == 401:
                 raise ServiceHandleException(msg="backup failed", msg_show="有状态组件必须停止方可进行备份")
+            if "some services do not exist in need backup services" in e.message["body"].get("msg", ""):
```

Comment:
How did this happen?

Reply:
The console component source code detection is completed, but you do not click Create, the backup will have this problem